sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -0,0 +1,263 @@
+         "@azure/test-utils-perfstress": "^1.0.0"
+       }
+    ```
+
+    _Note: `"@azure/test-utils-perfstress"` is not a published npm package._
+
+    Set the name of the package and mark it as private.
+
+    ```json
+     "name": "@azure-tests/perf-<service-sdk>",
+     "private": true,
+    ```
+
+5.  Run `rush update` and commit the changes to the `pnpm-lock` file.
+6.  Copy the `tsconfig.json`, `sample.env`(and `.env`) files that are present at the `sdk/<service>/<service-sdk>` to `sdk/<service>/perf-tests/<service-sdk>`.
+
+    TSCONFIG
+
+    - Modify the "extends" string in the copied tsconfig by adding ".." since the perf tests project is located a level below the actual SDK.
+    - Set the `compilerOptions.module` to `commonjs` in the `tsconfig` to allow running the tests with `ts-node`.
+
+    In the end, your tsconfig may look something like below.
+
+    ```
+        {
+          "extends": "../../../../tsconfig.package",
+          "compilerOptions": {
+            "module": "CommonJS",
+            "target": "ES2015",
+            "declarationDir": "./typings/latest",
+            "lib": ["ES6", "ESNext.AsyncIterable", "DOM"],
+            "noEmit": true
+          },
+          "compileOnSave": true,
+          "exclude": ["node_modules"],
+          "include": ["./test/**/*.ts"]
+        }
+    ```
+
+### [For perf-testing track 1 version of the same package](#For-perf-testing-track-1-version-of-the-same-package)
+
+(_Skip this section if your service does not have or does not care about a track-1 version._)
+
+1. If there is an old major version of your package that needs to be compared, create the folder as `sdk/<service>/perf-tests/<service-sdk>-track-1`
+
+2. It is expected that the track-1 perf tests are counterparts of track-2 tests, so they need to have the same names as specified in the track-2 tests for convenience.
+
+3. Add a `package.json` such as [example-track-1-perf-package.json](https://github.com/Azure/azure-sdk-for-js/blob/fe9b1e5a50946f53b6491d7f67b2420d8ee1b229/sdk/storage/perf-tests/storage-blob-track-1/package.json) at `sdk/<service>/perf-tests/<service-sdk>` folder.
+
+   Make sure to import your `<service-sdk>` and the `test-utils-perfstress` project.
+
+   ```json
+     "devDependencies": {
+        "@azure/<service-sdk>": "^<latest-track-1-version>",
+        "@azure/test-utils-perfstress": "file:../../../test-utils/perfstress/azure-test-utils-perfstress-1.0.0.tgz",
+      }
+   ```
+
+   Set the name of the package and mark it as private.
+
+   ```json
+    "name": "@azure-tests/perf-<service-sdk>-track-1",
+    "private": true,
+   ```
+
+   _Note: Track-1 packages will not be managed by `rush`, instead `npm` will be used to manage/run the track-1 tests, you can copy the readme such as the [storage-blob-perf-tests-track-1-readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/perf-tests/storage-blob-track-1/README.md) for instructions._
```

Comment:
but how will you get the perftest framework package without rush?

Reply:
The next point talks about the `npm run setup` which takes care of getting the perf framework for track-1 perf tests.
So I think I don't have to mention more details in this _Note_.